### PR TITLE
Update release notes for 2.0.4

### PR DESCRIPTION
**Pull request by bitcartel**

### CODE REVIEW

**Comment by bitcartel** [comment]
@zkbot r+

File: doc/release-notes.md [hunk 1]
```diff
@@ -4,3 +4,46 @@ release-notes at release time)
...
+Miners who wish to use the same address for every mined block, should use the
+`-mineraddress` option. 
+
+[Mining Guide](https://zcash.readthedocs.io/en/latest/rtd_pages/zcash_mining_guide.html)
+
+
+New consensus rule: Reject blocks that violate turnstile (Testnet only)
+-----------------------------------------------------------------------
+Testnet nodes will now enforce a consensus rule which marks blocks as invalid
+if they would lead to a turnstile violation in the Sprout or Shielded value
```
Comment (daira) [inline doc/release-notes.md]: Sprout or Sapling.

**Comment by bitcartel** [comment]
@zkbot r-

File: doc/release-notes.md (hunk 1 above)
Comment (bitcartel) [reply doc/release-notes.md]: Good spot. Will cancel the merge, push fix, etc.

**Comment by bitcartel** [comment]
Addressed @daira review comment.

**Comment by bitcartel** [comment]
@zkbot r+

**Pull request merged**